### PR TITLE
asciidoctor: add page

### LINKED ISSUE #6425: page request: asciidoctor
https://asciidoctor.org/docs/

### PR DESCRIPTION
**Pull request by glapa-grossklag**
Closes #6425.

- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
The descriptions of the examples shouldn't contain the file names, instead you might use something like `Convert an AsciiDoc file to [...]`.

File: pages/common/asciidoctor.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# asciidoctor
+
+> An AsciiDoc processor to read and translate AsciiDoc to a publishable format.
+> More information: <https://docs.asciidoctor.org>.
+
+- Convert `path/to/document.adoc` to HTML (the default output format):
+
+`asciidoctor {{path/to/document.adoc}}`
```
Comment (marchersimon) [inline pages/common/asciidoctor.md:8]:
```suggestion
- Convert a specific `.adoc` file to HTML (the default output format):

`asciidoctor {{path/to/file.adoc}}`
```
This is the format we usually use.

Comment (glapa-grossklag) [reply pages/common/asciidoctor.md]: Got it, thanks! :-)

File: pages/common/asciidoctor.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# asciidoctor
+
+> An AsciiDoc processor to read and translate AsciiDoc to a publishable format.
```
[the remark this answers is not in the document]
Comment (glapa-grossklag) [reply pages/common/asciidoctor.md:3]: Yes, I agree. What do you think about the following?
> Convert AsciiDoc files to a publishable format.

[the remark this answers is not in the document]
Comment (pixelcmtd) [reply pages/common/asciidoctor.md:3]: I don't like that. It doesn't describe the tool, but what you do with it, and just kinda sounds off to me.

[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/asciidoctor.md:3]: @nicokosi's suggestion seems the most logical to me

File: pages/common/asciidoctor.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`asciidoctor -a stylesheet={{path/to/stylesheet.css}} {{path/to/file.adoc}}`
+
+- Convert a specific `.adoc` file to embeddable HTML, removing everything except the body:
+
+`asciidoctor --embedded {{path/to/file.adoc}}`
+
+- Convert a specific `.adoc` file to a PDF using the `asciidoctor-pdf` library:
+
+`asciidoctor --backend=pdf --require=asciidoctor-pdf {{path/to/file.adoc}}`
```
Comment (marchersimon) [inline pages/common/asciidoctor.md:20]: Even though it's a fixed value in *this* example, it's still an argument and should go in token syntax:
```suggestion
`asciidoctor --backend={{pdf}} --require={{asciidoctor-pdf}} {{path/to/file.adoc}}`
```

**Review by marchersimon — APPROVED** [review]
Thanks @glapa-grossklag

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @glapa-grossklag!

**Review by pixelcmtd — APPROVED** [review]

**Review by nicokosi — COMMENTED** [review]
Well done! 👍

**Pull request merged**